packages/frontend/src/components/DocumentationHelpButton.tsx

```diff
@@ -9,32 +9,27 @@ type Props = React.AnchorHTMLAttributes<HTMLAnchorElement> &
         tooltipProps?: Partial<TooltipProps>;
     };
 
-const DocumentationHelpButton: FC<Props> = ({
-    iconProps,
-    tooltipProps,
-    ...anchorProps
-}) => (
+const DocumentationHelpButton: FC<Props> = ({ iconProps, tooltipProps }) => (
     <Tooltip
         withinPortal
         label="Open documentation"
         position="top"
         maw={350}
         {...tooltipProps}
     >
-        <Anchor
-            role="button"
+        <ActionIcon
+            component="a"
             target="_blank"
             rel="noreferrer"
-            color="dimmed"
-            {...anchorProps}
+            href="https://docs.lightdash.com/get-started/setup-lightdash/connect-project"
```

Comment:
I would remove this, and spread `{...anchorProps}` like it was before, so like this: 
```
<ActionIcon
            component="a"
            target="_blank"
            rel="noreferrer"
            {...anchorProps}
            >
            ```
            
            and ensure that above you destruct these:
            
            ```
const DocumentationHelpButton: FC<Props> = ({ iconProps, tooltipProps, ...anchorProps }) => (
            ```